src/Neo/SmartContract/Native/StdLib.cs

```diff
@@ -222,5 +223,39 @@ private static string[] StringSplit([MaxLength(MaxInputLength)] string str, stri
             StringSplitOptions options = removeEmptyEntries ? StringSplitOptions.RemoveEmptyEntries : StringSplitOptions.None;
             return str.Split(separator, options);
         }
+
+        [ContractMethod(CpuFee = 1 << 8)]
+        private static int StringByteLength([MaxLength(MaxInputLength)] string str)
```

Comment:
It is same as `OpCode.SIZE`?

Reply:
> It is same as `OpCode.SIZE`?

Exactly

Reply:
> > It is same as `OpCode.SIZE`?
> 
> Exactly, just thought it would be better to make the naming style consistent.

Is not the same because this take in care about the unicode chars